self-core-impl/src/test/java/com/selfxdsd/core/mock/InMemoryTasksTestCase.java

```diff
@@ -37,6 +37,8 @@
  * @todo #135:30min Finish 'getTasksOfContributor()' test case
  *  when API permits that; as in when a Task can be assigned to a
  *  Contributor.
+ * @todo #439:30min Test assign and unassign Task using the in-memory
+ *  implementation of {@link Storage}.
```

Comment:
@criske You don't really need to use the in-memory storage mock. You can test Task.unassign(...) by just using Mockito :D

Reply:
@amihaiemil I've thought it would be nice to write it as an "integration" test, now that we have almost all in memory pieces implemented. But sure with mocks will be enough.
Now that #437 is implemented it's easier to write the test. Assign/unassign is using Task#issue() to get the id, which is a network call. Having issue id available will make things much better.
Should I update with the #437 PR and write the test, or just it leave as a puzzle?

Reply:
@criske Just leave it as a puzzle :D